sdk/identity/azure-identity/azure/identity/_internal/linux_vscode_adapter.py

```diff
@@ -59,10 +59,10 @@ def _get_user_settings():
     try:
         with open(path) as file:
             data = json.load(file)
-            environment_name = data.get("azure.cloud", "Azure")
+            environment_name = data.get("azure.cloud", "AzureCloud")
```

Comment:
The change looks fine. 
Shouldn't this env also alter the target Azure Auth Host for VS code client ?
the auth host provided via kwargs/constructor/env will take priority, but if isn't specified then this env should be used to target a specific auth host too.

Reply:
Perhaps; that would be a new feature. I put it on the backlog as #14808